src/components/DatePickerCalendar.jsx

```diff
@@ -1,94 +1,29 @@
-import React, { useState } from 'react';
-import { Modal, TouchableOpacity, View, StyleSheet,Text } from 'react-native';
-import DatePicker from 'react-native-modern-datepicker'
-import { getToday, getFormatedDate } from 'react-native-modern-datepicker';
-
-
-
-const DatePickerCalendar = () => {
-  const today = new Date ();
-  const startDate = getFormatedDate(today.setDate(today.getDate() + 1), 'YYYY/MM/DD');
-
-  const [open, setOpen] = useState(false)
-  const [date, setDate] = useState('')
-  
-  function handleOnPress(){
-   setOpen(!open);
-  }
-
-  function handleChange(propDate) {
-    setDate(propDate);
-    console.log(propDate);
-  }
-  
-
+import React from 'react';
+import {View} from 'react-native';
+import DatePicker from 'react-native-modern-datepicker';
+import moment from 'moment';
+import DatePickerCalendarStyles from '../styles/DatePickerCalendarStyles';
+
+const DatePickerCalendar = ({handleChange, date}) => {
+  const today = new Date();
+  today.setDate(today.getDate() + 1);
+  const startDate = moment(today).format('YYYY-MM-DD');
   return (
-	
-	<View style={styles.container}>
-
-	<TouchableOpacity onPress={handleOnPress}>
-	<Text>Open</Text>
-	</TouchableOpacity>
-	
-	
-	<Modal
-	animationType="slide"
-	transparent={true}
-	visible={open}
-	>
-	
-	<View style={styles.centeredView}>
-	<View style={styles.modalView}>
-	
-	<DatePicker
-  options={{
-    mainColor: '#6172F6',
-    textFontSize: 17,
-    textHeaderFontSize: 20,
-    selectedTextColor: '#fff',
-  }}
-	mode='calendar'
-	minimumDate={startDate}
-	selected={date}
-	onSelectedChange={handleChange}
-	 />
-	
-
-	<TouchableOpacity onPress={handleOnPress}>
-	<Text>Close</Text>
-	</TouchableOpacity>
-
-	</View>
-	</View>
-    </Modal>
+    <View style={DatePickerCalendarStyles.modalView}>
+      <DatePicker
+        options={{
+          mainColor: '#6172F6',
+          textFontSize: 17,
+          textHeaderFontSize: 20,
+          selectedTextColor: '#fff',
```

Comment:
If you have a color library, you can use it here as well.

The styles must go in an external file.